appointment/upload-documents/uploadDocuments.php

```diff
@@ -143,8 +144,8 @@ class="dcf-btn dcf-btn-secondary"
 				<fieldset class="dcf-mt-3">
 					<legend>Appointment Ready</legend>
 					<p>
-						After you mark your appointment as 'ready', a tax preparer will begin preparing your return within 48 hours. 
-						If the tax preparer does not have all the necessary documents, they will not be able to prepare your return. 
+						Thank you for uploading your documents. Your appointment is {{appointmentTimeStr}}.
```

Comment:
I'm leery to say "Thank you for uploading your documents" in this box since this box is displayed all the time. I think a description of what "is ready" means is more valuable (i.e. "all your documents have been uploaded").

Also, the "Your documents need to be uploaded by DEADLINE (one week in advance)" text--maybe we should move that out of this box (or duplicate it outside of this box), somewhere a little more noticeable, like at the top of the page or  something? This deadline seems like something pretty important

Reply:
I thought the same thing! Tracy sent me that wording, but I offered the same feedback and suggested we omit the thanks and switch the box name to say "Finished uploading your documents?" Hopefully this will be a bit clearer. I think duplicating that text sounds like a good idea, too.